src/main/java/com/google/cloud/genomics/dataflow/pipelines/CountReads.java

```diff
@@ -130,9 +141,25 @@ public void processElement(DoFn<Long, String>.ProcessContext c) throws Exception
       }
     }).named("toString"));
     readCountText.apply(TextIO.Write.to(options.getOutput()).named("WriteOutput"));
+
     p.run();
   }
 
+  private static boolean GCSURLExists(String url) {
```

Comment:
Yes, I've been meaning to do a pass at the storage stuff but haven't yet.
This method is a bit thin to make its own class, and it's not really a
perfect fit onto GCSOptions. Didn't we have a GCSDownloader somewhere?
There are a bunch of helper functions in this repo and the pipeline ones
that we should unify. I'm not sure now's the time to do it, though.

On Thu, Apr 16, 2015 at 3:46 PM, Nicole Deflaux <notifications@github.com>
wrote:

> In
> src/main/java/com/google/cloud/genomics/dataflow/pipelines/CountReads.java
> <https://github.com/googlegenomics/dataflow-java/pull/55#discussion_r28557015>
> :
>
> >      p.run();
> >    }
> >
> > +  private static boolean GCSURLExists(String url) {
>
> This is a generally useful method. If its not already available in a GCS
> java library we depend upon, it would be better in
> com.google.cloud.genomics.dataflow.util
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/googlegenomics/dataflow-java/pull/55/files#r28557015>.
>